mmn/mmn.go

```diff
@@ -0,0 +1,77 @@
+package mmn
+
+import (
+	"github.com/mysteriumnetwork/node/core/connection"
+	"github.com/mysteriumnetwork/node/core/service"
+	"github.com/mysteriumnetwork/node/eventbus"
+	"github.com/mysteriumnetwork/node/identity"
+
+	"github.com/rs/zerolog/log"
+)
+
+type MMN struct {
+	collector *Collector
+	client    *client
+	eventBus  eventbus.EventBus
+}
+
+func NewMMN(collector *Collector, client *client, eventBus eventbus.EventBus) *MMN {
+	return &MMN{collector, client, eventBus}
+}
+
+func (m *MMN) SubscribeToEvents() {
```

Comment:
accept eventBus as a param here, remove from MMN struct as you're already calling a method explicitly for subscription there is no benefit of having it attached to the struct